recipe/meta.yaml

```
@@ -31,29 +30,34 @@ requirements:
     - wheel
     - llvmdev 11.1.*
     - llvm 11.1.*
-    - zlib
-    - vs2015_runtime  # [win]
+    - zlib 1.2.*
+    - vs2015_runtime 14.16.27012 # [win]
   run:
     - python
     - zlib
     - vs2015_runtime  # [win]
 
 test:
+  requires:
+    - pip
   imports:
     - llvmlite
     - llvmlite.binding
```

```suggestion
    - llvmlite.binding
    - llvmlite.ir
    - llvmlite.tests
```
https://github.com/numba/llvmlite/blob/v0.40.0/setup.py#L180

---

done